docs/tutorials/optimizers_movingaverage.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Could you leave the cell results visible so it is easier to follow without having to run the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>